src/strategies/primitiveChecks/bitUsed.ts

```diff
@@ -0,0 +1,38 @@
+import isDeployed from "./isDeployed"
+import bitmapPointer from '../../utils/bitmapPointer'
+import BigNumber from 'bignumber.js'
+import { padLeft } from 'web3-utils'
+
+
+// if bit is used, return false, else return true
+const bitUsed = async (provider: any, accountAddress: String, bitmapIndex: BigInt, bit: BigInt) => {
```

Comment:
I think `provider` can be `providerOrSigner` and use the Ethers type ProviderOrSigner

and the bigUsed function should have an explicit return type of `boolean`

Reply:
here's an example of using signer or provider https://github.com/brinktrade/brink-sdk/blob/v5/src/oracles/readUint256Oracle.ts#L5